docs/API-functions.md

```diff
@@ -1182,7 +1182,7 @@ when you are allowed to send or retrieve a part of the message.
 (specified as 0), the current system time (absolute microseconds since epoch) is 
 used.
    * [OUT] Receiver only. Specifies the time when the packet was intended to be
-delivered to the receiver.
+delivered to the receiver (microseconds since epoch).
```

Comment:
What "epoch"? Changing the system time changes the machine's epoch, and steady clock has a different epoch than system clock. I think you should here make at best a reference to where it is described in details.